jdaviz/configs/cubeviz/plugins/tools.py

```diff
@@ -59,49 +57,9 @@ def on_mouse_event(self, data):
 
 
 @viewer_tool
-class SpectrumPerSpaxel(CheckableTool):
+class SpectrumPerSpaxel(SinglePixelRegion):
 
     icon = os.path.join(ICON_DIR, 'pixelspectra.svg')
     tool_id = 'jdaviz:spectrumperspaxel'
```

Comment:
did we want to change the icon in cubeviz to be the same?  Should that tool also be nested in the subset menu or does it "deserve" to have a dedicated place on the toolbar because it'll be used that much?

Reply:
The Cubeviz icon and placement got a lot of discussions back when it was implemented. So, I'd rather not re-open that can of worms but I'll ping @javerbukh just in case.